### DISCUSSION TITLE
Why is there issues after level 8? (-1)

### DISCUSSION THREAD

**Discussion by orklah** [vimeo/psalm#5065] [Ideas]
It is very inconvenient not to be able to choose to suppress those with errorLevel. When creating a plugin to show only specific issues we have to list every issue in config file to be sure to suppress any non-relevant issue (@klebann asked about this a few weeks ago and I had to do it myself for a plugin I'm creating).

Couldn't we just put those in level 9?

**Comment by muglug** [comment]
I think a better option would be an explicit mode where Psalm only emits certain issues. This could be done via a command line `--issues=X,Y` flag or some sort of plugin API

**Reply by orklah** [reply]
Interesting. This would need to work with Plugin issues also. I'll think about this, thanks!
